src/main/clojure/starfish/core.clj

```diff
@@ -312,16 +312,17 @@
       :else (throw (IllegalArgumentException. (str "Invalid did: " (class agent-did)))))))
 
 (defn digest
-  "Computes the keccak256 hash of the byte representation of some data and returns this as a hex string.
+  "Computes the sha3_256 String hash of the byte representation of some data and returns this as a hex string.
 
   Handles
    - byte arrays - hashed as-is
    - Strings - converted to UTF-8 representation
-   - Assets - gets content hash
+   - Assets - compute the hash of asset content
```

Comment:
Presume this should be hash of asset metadata

Reply:
updated